cmd/manager/main.go

```diff
@@ -57,7 +57,7 @@ func init() {
 	flagset.StringVar(&flagPluginsInitContainerImage, "grafana-plugins-init-container-image", "", "Overrides the default Grafana Plugins Init Container image")
 	flagset.StringVar(&flagPluginsInitContainerTag, "grafana-plugins-init-container-tag", "", "Overrides the default Grafana Plugins Init Container tag")
 	flagset.StringVar(&flagNamespaces, "namespaces", "", "Namespaces to scope the interaction of the Grafana operator. Mutually exclusive with --scan-all")
-	flagset.BoolVar(&scanAll, "scan-all", false, "Scans all namespaces for dashboards")
+	flagset.BoolVar(&scanAll, "scan-all", true, "Scans all namespaces for dashboards")
```

Comment:
```the --scan-all flag is not set by default. By default the operator only scans for dashboards in its own namespace.```

@pb82 regarding your comment, should this be set to false in that case?

Reply:
Oh I see, this has changed. Was there a specific reason for this change? This would affect OLM installations because they currently ship with this flag set to false (the old default).

Reply:
Thinking about it now, I think I'll revert this change and it's probably safer to have it as `False` to prevent things such as OLM breaking, AFAIK this arg can be overwritten in the yaml( Which integreatly already does, but limits these namespaces to just RHMI ones)   https://github.com/integr8ly/application-monitoring-operator/blob/master/templates/grafana-operator.yaml#L23